apps/client/src/common/components/schedule-time/ScheduleTime.scss

```
@@ -0,0 +1,27 @@
+@use '@/theme/viewerDefs' as *;
+
+.schedule__ {
+  padding-top: 0.5rem;
+
+  display: flex;
```

Stylelint (17.14.0)

[error] Expected no empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)